core/style/style.view.js

```diff
@@ -110,11 +110,20 @@ export class StyleView extends View {
 					const cells = bodyRow.cells();
 					for (let j = 0, cellsLength = cells.length; j < cellsLength; j++) {
 						const cell = cells[j];
-						const column = columns[j];
+						let columnIdex = -1;
```

Comment:
why u need all that code?

Reply:
This code fixes a bug that caused that the column chooser didn't work. When some columns are filtered, they are removed from the columns array. The cells array knows nothing about it and won't be changed. So the indexes of the cells and corresponding columns are not the same anymore. This caused the issue I mentioned above.

Reply:
fixed (reverted back)